dotcom-rendering/src/types/configContext.ts

```diff
@@ -11,10 +11,12 @@ export type Config =
 	| {
 			renderingTarget: Extract<RenderingTarget, 'Web'>;
 			darkModeAvailable: boolean;
+			inAdvertisingPartnerABTest: boolean;
```

Comment:
This could be optional and reduce the surface area of this PR a bit! All the modified tests etc.

Reply:
@arelra and I had a talk about it and Ravi thought it would be better to have it as mandatory and for the tests it will be removed anyways when we remove the AB test. Maybe I need to understand what are the pros and cons if we do it as mandatory or optional?

Reply:
I guess it does make it much easier to tell if you've included it everywhere you need! Probs best to leave it👍